core/src/main/java/feast/core/job/UpgradeJobTask.java

```diff
@@ -22,6 +22,7 @@
 import lombok.Getter;
 import lombok.Setter;
 
+/** Task that upgrade given {@link Job} by restarting it in {@link JobManager} */
```

Comment:
Is there a difference between an update and an upgrade, and what does an `upgrade` actually mean?

Reply:
I was looking for separation between update status (pooling smth) and upgrade job (pushing changes). Naming smth is hardest programming task

Reply:
Yea that make sense, but this method is called `upgradeJob` but it calls `updateJob`, which is why I asked. Should updateJob be renamed as well? Not sure I feel too strongly on this, just asking.